lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -1862,10 +1859,10 @@ LogicalResult MemOp::verify() {
     // found.
     FIRRTLBaseType dataType;
     if (portKind == MemOp::PortKind::Debug) {
-      auto resType = getResult(i).getType().dyn_cast<RefType>();
+      auto resType = firrtl::type_dyn_cast<RefType>(getResult(i).getType());
       if (!(resType && resType.getType().isa<FVectorType>()))
         return emitOpError() << "debug ports must be a RefType of FVectorType";
-      dataType = resType.getType().cast<FVectorType>().getElementType();
```

Comment:
This is stripping the alias to see if the port is a reftype<VectorType> -- should it strip the type the reference is "to" as well?

Since this is only something we create maybe neither matters, but not sure why one and not the other.